index.html

```diff
@@ -27,17 +27,119 @@ <h1>I'm Saningo Glad to see you</h1>
         <section class="social-details">
           <h5>LET'S CONNECT</h5>
           <div class="social-wrapper">
-            <span class="social-icon"><img src="/images/Vector.svg" alt="social-icon"/></span>
-            <span class="social-icon"><img src="./images/Vector.png" alt="social-icon" /><img alt="social-icon" src="/images/Vect-1.png"/></span>
-            <span class="social-icon"><img src="./images/Vect-1.svg" alt="social-icon"/></span>
-            <span class="social-icon"><img src="./images/Vect-2.svg" alt="social-icon" /></span>
-            <span class="social-icon"><img src="./images/Vect-3.svg" alt="social-icon" /></span>
+            <ul>
+              <li class="social-icon"><img src="/images/Vector.svg" alt="social-icon"/></li>
+              <li class="social-icon"><img src="./images/Vector.png" alt="social-icon" /><img alt="social-icon" src="/images/Vect-1.png"/></li>
+              <li class="social-icon"><img src="./images/Vect-1.svg" alt="social-icon"/></li>
+              <li class="social-icon"><img src="./images/Vect-2.svg" alt="social-icon" /></li>
+              <li class="social-icon"><img src="./images/Vect-3.svg" alt="social-icon" /></li>
+            </ul>
           </div>
         </section>
+        <!-- Projects Section -->
+        <section class="work">
+          <div class="card">
+            <div class="card-container">
+              <div class="card-1"></div>
+              <div class="card-desc">
+                <h2>Tonic</h2>
+                <ul>
+                  <li>Canopy</li>
+                  <li>Back End Dev</li>
+                  <li>2022</li>
+                </ul>
+                <div class="desc">
+                  <p>A daily selection of privately personalized reads, no accounts or sign-ups required</p>
+                  <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                  </ul>
+                  <button type="button">See Project</button>
+                </div>
```

Comment:
- [ ] Let's change the design as given in the requirement. While changing be careful about font size, style, etc.
- Your website
![nty](https://user-images.githubusercontent.com/26767383/153622188-3752cc6f-5173-4df6-800e-a26b63733a9f.PNG)
- Original website 
![hrtgrer](https://user-images.githubusercontent.com/26767383/153622291-90021a20-52a9-4109-8d85-2f756edb4db9.PNG)

- [ ] Kindly use the above pic as a reference and fix others tile content also.